Makefile

```diff
@@ -255,9 +255,11 @@ LDFLAGS ?= $(shell hack/version.sh)
 # Allow overriding manifest generation destination directory
 MANIFEST_ROOT ?= ./config
 CRD_ROOT ?= $(MANIFEST_ROOT)/default/crd/bases
-SUPERVISOR_CRD_ROOT ?= $(MANIFEST_ROOT)/supervisor/crd
+SUPERVISOR_CRD_ROOT ?= $(MANIFEST_ROOT)/supervisor/crd/bases
```

Comment:
wondering about why we have all this changes in Makefile and move the crds around 🤔 

Note: I'm okay with it because it aligns to what we have for govmomi mode :-) just want to understand

(except the controller-gen)

Reply:
Sorry for the late response, just back to working on this :)

No specific reasons actually, as this PR adds more manifests and kustomization files under `/supervisor`, so need to make it more structured.